pkg/secretless/plugin/so/available_plugins_test.go

```diff
@@ -34,7 +35,7 @@ func getMockPlugins() AvailablePlugins {
 
 func TestPlugins(t *testing.T) {
 	t.Run("HTTPPlugins", func(t *testing.T) {
-		httpPlugins := getMockPlugins().HTTPPlugins()
+		httpPlugins := HTTPPlugins()
```

Comment:
Let me look into this later, this wasn't code I wrote.  This just got folded into the PR because of ripple changes from the "Move to..." refactoring the IDE did.